protocol/messenger.go

```diff
@@ -3678,6 +3678,48 @@ func (m *Messenger) saveDataAndPrepareResponse(messageState *ReceivedMessageStat
 			}
 		}
 
+		if installation.Enabled {
+			// // Delete AC notif since the installation is now enabled
```

Comment:
Actually, are we sure we want to delete the notification, no like `Accept` it or smth?

Reply:
As we wish. I think the notification serves no further purpose once accepted, but it's as we prefer.